packages/app/src/search/SearchTab.tsx

```diff
@@ -0,0 +1,50 @@
+import { getNameFromPath } from '../providers/utils';
+import SearchEntry from './SearchEntry';
+import styles from './SearchTab.module.css';
+
+interface Props {
+  paths: string[];
+  value: string;
+  onChange: (newValue: string) => void;
+  selectedPath: string;
+  onSelect: (path: string) => void;
+}
+
+function SearchTab(props: Props) {
+  const { paths, value, onChange, selectedPath, onSelect } = props;
+
+  const matches = value
+    ? paths.filter((path) => getNameFromPath(path).includes(value))
```

Comment:
The search algorithm is very basic for now: get the name of the entity and see if it includes the pattern